orchard-provider-aws/src/main/scala/com/salesforce/mce/orchard/io/aws/activity/EmrActivity.scala

```diff
@@ -68,7 +68,7 @@ case class EmrActivity(name: String, steps: Seq[EmrActivity.Step], clusterId: St
     } else if (statuses.contains(StepState.FAILED)) {
       Status.Failed
     } else if (statuses.contains(StepState.CANCELLED)) {
-      Status.Canceled
+      Status.Failed
```

Comment:
why it should map to failed?

Reply:
@schowsf probably worth adding a comment here, in case we are asking the same question again in the future.

Reply:
@anniezhang77, @realstraw, good idea.  Comment added.